c2rust-analyze/src/context.rs

```diff
@@ -137,6 +140,7 @@ impl<'tcx> GlobalAnalysisCtxt<'tcx> {
             tcx: _,
             lcx,
             ref mut fn_sigs,
+            field_tys: _,
```

Comment:
`field_tys` need to be relabeled like `fn_sigs` here.

Reply:
let me know if 7bf10839 doesn't address this correctly

Reply:
The `fn_sigs` case looks like this:
```Rust
sig.output = remap_lty_pointers(lcx, map, sig.output);
```
So I think `remap_lty_pointers` just returns the remapped `LTy`, and you have to assign it back into `*labeled_field`.

Reply:
addressed in 6b91f4bf